lib/graph3d/Graph3d.js

```diff
@@ -67,6 +67,29 @@ var DEFAULTS = {
 
   style            : Graph3d.STYLE.DOT,
   tooltip          : false,
+
+  tooltipStyle     : {
```

Comment:
Tested all combos, if no `tooltipStyle` is provided the tooltips are not changed and are exactly as before.